README.md

```diff
@@ -460,6 +462,11 @@ if you run into any errors or problems with the prebuilt wheels.
 See [Building JAX from
 source](https://jax.readthedocs.io/en/latest/developer.html#building-from-source).
 
+## Neural network libraries
+
+There are lots of great deep learning libraries built on top of JAX!
```

Comment:
This sentence, and especially the `lots` part, can be interpreted in different ways by different people. I suggest skipping this intro and going directly to the descriptions.